### ISSUE TITLE
Separate Dual Icons & Custom Keybinds Don't Work and Causes GD to Freeze

### ISSUE THREAD

**Issue by kippp-fowl** [Weebifying/separate-dual-icons-geode#14]
I installed this mod earlier today because the concept was pretty interesting, only to find out that GD would constantly freeze when I press keys especially the escape key which is super annoying because I use that to exit levels. I originally thought that the mod was not working, but I decided to remove all other mods and keep this mod with the node ids enabled to find out that the mod works perfectly fine. I then enabled the custom keybinds mod to find out that it was the source of the crash. I don't know if it is an OS-related bug or something else, but if you can replicate that issue, then that can be very useful. I have tried using this mod with all of my other mods and it seems to be working. I need the custom keybinds mod to have the clicks between frames mod so I'm not removing it. If video footage is necessary, then I am happy to give it to you.

TL:DR: Using both the separate dual icons mod and the custom keybinds mod will cause my game to freeze after I press any key, forcing me to force quit GD. If you can replicate and find a solution to this issue, that would be greatly appreciated.

Thank you.

**Comment by Weebifying** [comment]
Are you sure my mod and Custom Keybinds clashing and not just CK's problem? Cuz this just sounds absolutely ridiculous and I can't replicate it. Can you send a video of you replicating the crash and also the crashlog?

**Comment by wquasar5** [comment]
If you can't replicate it at all, then there's definitely some other problem maybe with my OS (Linux).

Video below:
https://github.com/Weebifying/separate-dual-icons-geode/assets/124114934/11cc3042-ae5e-43d2-b80d-584fca5eb761

I pressed on the w key to jump and gd just freezes. There's no crash at all as nothing new shows up in my crash logs. Maybe the logs of my current session can help, but other than that, I don't know.

[Geode 2024-06-28 12.36.27.log](https://github.com/user-attachments/files/16033717/Geode.2024-06-28.12.36.27.log)
